charts/devtron/devtron-bom.yaml

```diff
@@ -28,23 +28,28 @@ components:
         HIDE_EXCLUDE_INCLUDE_GIT_COMMITS: "false"
         ENABLE_SCOPED_VARIABLES: "true"
         ENABLE_CI_JOB: "true"
-        GLOBAL_API_TIMEOUT: "60000"
+        GLOBAL_API_TIMEOUT: "6000000"
```

Comment:
why we increased the timeout

Reply:
I used the values which was specified on cd have reverted to default.